src/options.c

```diff
@@ -260,120 +260,120 @@ static struct opt_entry opt_list[] = {
 "                           format)\n" \
-"--single-pair-max=N        override max. number of word pairs generated (%u)\n" \
-"--no-single-pair           disable single word pair generation\n" \
-"--[no-]single-retest-guess override config for SingleRetestGuess\n" \
-"--wordlist[=FILE] --stdin  wordlist mode, read words from FILE or stdin\n" \
+"--single-pair-max=N        Override max. number of word pairs generated (%u)\n" \
+"--no-single-pair           Disable single word pair generation\n" \
+"--[no-]single-retest-guess Override config for SingleRetestGuess\n" \
+"--wordlist[=FILE] --stdin  Wordlist mode, read words from FILE or stdin\n" \
 "                  --pipe   like --stdin, but bulk reads, and allows rules\n" \
-"--rules[=SECTION[,..]]     enable word mangling rules (for wordlist or PRINCE\n" \
+"--rules[=SECTION[,..]]     Enable word mangling rules (for wordlist or PRINCE\n" \
 "                           modes), using default or named rules\n" \
-"--rules=:rule[;..]]        same, using \"immediate\" rule(s)\n" \
-"--rules-stack=SECTION[,..] stacked rules, applied after regular rules or to\n" \
+"--rules=:rule[;..]]        Same, using \"immediate\" rule(s)\n" \
+"--rules-stack=SECTION[,..] Stacked rules, applied after regular rules or to\n" \
 "                           modes that otherwise don't support rules\n" \
-"--rules-stack=:rule[;..]   same, using \"immediate\" rule(s)\n" \
-"--rules-skip-nop           skip any NOP \":\" rules (you already ran w/o rules)\n" \
-"--loopback[=FILE]          like --wordlist, but extract words from a .pot file\n" \
-"--mem-file-size=SIZE       size threshold for wordlist preload (default %u MB)\n" \
-"--dupe-suppression         suppress all dupes in wordlist (and force preload)\n" \
-"--incremental[=MODE]       \"incremental\" mode [using section MODE]\n" \
-"--incremental-charcount=N  override CharCount for incremental mode\n" \
-"--external=MODE            external mode or word filter\n" \
-"--mask[=MASK]              mask mode using MASK (or default from john.conf)\n" \
+"--rules-stack=:rule[;..]   Same, using \"immediate\" rule(s)\n" \
+"--rules-skip-nop           Skip any NOP \":\" rules (you already ran w/o rules)\n" \
+"--loopback[=FILE]          Like --wordlist, but extract words from a .pot file\n" \
+"--mem-file-size=SIZE       Size threshold for wordlist preload (default %u MB)\n" \
+"--dupe-suppression         Suppress all dupes in wordlist (and force preload)\n" \
+"--incremental[=MODE]       \"Incremental\" mode [using section MODE]\n" \
+"--incremental-charcount=N  Override CharCount for incremental mode\n" \
+"--external=MODE            External mode or word filter\n" \
+"--mask[=MASK]              Mask mode using MASK (or default from john.conf)\n" \
 "--markov[=OPTIONS]         \"Markov\" mode (see doc/MARKOV)\n" \
-"--mkv-stats=FILE           \"markov\" stats file (see doc/MARKOV)\n" \
+"--mkv-stats=FILE           \"Markov\" stats file (see doc/MARKOV)\n" \
```

Comment:
Scope creep: Now that `--markov` and `--mkv-stats` are together, we may drop the repat of "(see doc/MARKOV)" for the latter